src/components/newContact/NewContact.tsx

```diff
@@ -1,91 +1,98 @@
 import React, { FC, useState } from 'react'
 import styles from './newContact.module.scss'
 import { useDispatch, useSelector } from 'react-redux'
-import { selectIsLoading } from '../../redux/contacts/selectors'
+import { selectIsLoading, selectNewContact } from '../../redux/contacts/selectors'
+// import { Contact } from '../../models/interfaces/contact'
 
 // Components
 import Header from '../header/Header'
 import { setNewContact } from '../../redux/contacts/actions'
+import { FormEvent } from 'hoist-non-react-statics/node_modules/@types/react'
 
 const NewContact: FC = (props) => {
   const loading = useSelector(selectIsLoading);
+  const newContact = useSelector(selectNewContact);
 
   const dispatch = useDispatch();
-  const [contactInfo, setContactInfo] = useState({
-    firstName: '',
-    lastName: '', 
-    phone: '',
-    email: '',
-    address1: '',
-    address2: '',
-    city: '',
-    state: '',
-    zip: ''
-  })
+  // const [contactInfo, setContactInfo] = useState({
+  //   firstName: '',
+  //   lastName: '', 
+  //   phone: '',
+  //   email: '',
+  //   address1: '',
+  //   address2: '',
+  //   city: '',
+  //   state: '',
+  //   zip: ''
+  // })
 
-  const handleChange = (e: React.ChangeEvent<HTMLInputElement>) => {
-    e.persist();
-    setContactInfo((prevState) => ({
-      ...prevState,
-      [e.target.name]: e.target.value,
-    }));
-  };
+  // const handleChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+  //   e.persist();
+  //   setContactInfo((prevState) => ({
+  //     ...prevState,
+  //     [e.target.name]: e.target.value,
+  //   }));
+  // };
+
+  const handleSave = (e: FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    const { firstName, lastName, phone, email, address1, address2, city, state, zip } = e.target as typeof e.target & {
```

Comment:
I think it'd be cleaner to do all of this in redux. Each form item would have an onChange that modifies it's specific property on that object and then when your create thunk hits the fulfilled status you can clear the object out